core/src/main/scala/org/scalarelational/column/ColumnOption.scala

```diff
@@ -11,6 +12,6 @@ case class ColumnOption[T, S](column: ColumnLike[T, S]) extends ColumnLike[Optio
   def dataType: DataType[Option[T], S] = OptionDataTypeCreator.create[T, S](column.dataType)
   def manifest: Manifest[Option[T]] = column.manifest.asInstanceOf[Manifest[Option[T]]]
```

Comment:
Unfortunately we currently do for detecting "Option".  I'm not sure if there's any other use anymore.